Sources/SwiftInfo/main.swift

```diff
@@ -64,5 +95,5 @@ source.setEventHandler {
 ////////
 
 source.resume()
-Main.run()
+SwiftInfo.main(CommandLine.arguments)
```

Comment:
```suggestion
SwiftInfo.main()
```

This should work without the args, AFAIK.